setup.py

```diff
@@ -1,8 +1,10 @@
 from setuptools import setup, find_packages
 
-import imp
+import importlib.util
 
-version = imp.load_source('pyrubberband.version', 'pyrubberband/version.py')
+spec = importlib.util.spec_from_file_location('pyrubberband.version', 'pyrubberband/version.py')
+version = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(version)
```

Comment:
This whole thing can be simplified quite a bit by migrating to setup.cfg.  Resampy has a reasonable example of how this should look, and we can bypass all the imp/importlib business:
https://github.com/bmcfee/resampy/blob/main/setup.cfg

Reply:
Thanks for the reference! Migration to [setup.cfg](https://github.com/bmcfee/pyrubberband/pull/29/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52) is done. 
Tested on `python 3.6` ~ `python 3.12` and confirmed that all works fine.